### PR TITLE
Feat/formatting

### PR DESCRIPTION
**Pull request by b-d1**
## Pull request checklist

Please check if your PR fulfills the following requirements:

-   [x] Tests for the changes have been added (for bug fixes / features)
-   [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
-   [x] Build (`yarn build`) was run locally and any changes were pushed
-   [x] Lint (`yarn lint`) has passed locally and any fixes were made for failures

## Pull request type

Please check the type of change your PR introduces:

-   [x] Bugfix
-   [ ] Feature
-   [ ] Code style update (formatting, renaming)
-   [x] Refactoring (no functional changes, no api changes)
-   [ ] Build related changes
-   [ ] Documentation content changes
-   [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

- This PR includes changes to the name of the fields for the data received for webhook execution
-
-

## Does this introduce a breaking change?

-   [x] Yes
-   [ ] No

### CODE REVIEW

**Review by trozler — COMMENTED** [review]
Nicely done sir, just a minor comment.

File: src/utils/index.ts [hunk 1]
```diff
@@ -15,9 +15,14 @@ export const isValidAddress = (address: string): boolean => {
...
 export const isValidFlowRate = (flowRate: number): boolean => {
-    if (typeof flowRate !== "number") return false;
-    if (flowRate < 0) return false;
-    return true;
+    if (typeof flowRate !== "string") return false;
+    if (flowRate === "") return false;
+    try {
+        if(parseInt(flowRate) < 0) return false;
+        return true;
+    } catch (e) {
```
[the remark this answers is not in the document]
Comment (b-d1) [reply src/utils/index.ts:23]: Good question, but at this point it serves more as a sanity check only. Also we don't want to introduce BN and that kind of stuff in order not to bloat the library (cross-fetch is the only dependency as of right now).

[the remark this answers is not in the document]
Comment (trozler) [reply src/utils/index.ts:23]: Are flowRates stored in 18 decimal in string format?
If yes will parseInt function not fail a lot of the time by overflow?

[the remark this answers is not in the document]
Comment (b-d1) [reply src/utils/index.ts]: You are right fix has been pushed addressing this issue (thank you for the implementation btw).

**Review by trozler — APPROVED** [review]
lgtm

**Pull request merged**